docs-src/modules/ROOT/nav.adoc

```diff
@@ -1,10 +1,10 @@
 * xref:installing.adoc[Installing]
 * xref:loading-data.adoc[Loading Data]
 * Examples
-** https://github.com/kaskada-ai/kaskada/blob/main/examples/Kaskada%20Demo%20for%20Event%20Processing%20and%20Time-centric%20Calculations.ipynb[Overview Notebook]
-** https://github.com/kaskada-ai/kaskada/blob/main/examples/ML_example.ipynb[Real-time ML Notebook]
-** https://github.com/kaskada-ai/kaskada/blob/main/examples/Customer_Retention_(OSS).ipynb[Customer Retention Notebook]
-** https://github.com/kaskada-ai/kaskada/blob/main/examples/pulsar-materialize/notebooks/Materialize_to_Pulsar.ipynb[Materializing to Pulsar Notebook]
+** https://github.com/kaskada-ai/kaskada/blob/main/examples/Kaskada%20Demo%20for%20Event%20Processing%20and%20Time-centric%20Calculations.ipynb[Overview ↪]
```

Comment:
Maybe it is the way Github renders this text but are we adding a symbol for each link?

Reply:
Yeah, I want a way to distinguish "internal" and "external" links. There's no unicode for "external link", so I just picked this little arrow thing (our old docs used "↗").